os_stub/mbedtlslib/include/mbedtls/config.h

```diff
@@ -3996,6 +4032,14 @@ extern int my_snprintf(char *str, size_t size, const char *format, ...);
  */
 /*#define MBEDTLS_ECDH_VARIANT_EVEREST_ENABLED*/
 
+/**
+ * Allow library to access its structs' private members.
+ *
+ * Although structs defined in header files are publicly available,
+ * their members are private and should not be accessed by the user.
+ */
+#define MBEDTLS_ALLOW_PRIVATE_ACCESS
```

Comment:
Would you please explain why this is needed now?

Reply:
Migration guide [1] has a recommendation on how to access struct private fields. Based on that I have patch that will help to remove MBEDTLS_ALLOW_PRIVATE_ACCESS from config.h. I will soon post the changes for review.

[1] https://github.com/Mbed-TLS/mbedtls/blob/development/docs/3.0-migration-guide.md#most-structure-fields-are-now-private

Reply:
Thank you!